package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node build/index.js",
-    "dev": "export DEBUG=dev && nodemon --exec babel-node server/index.js",
+    "dev": "export DEBUG=dev NODE_ENV=development v&& nodemon --exec babel-node server/index.js",
```

Comment:
@encodedBicoding it's not clear to me what you are trying to achieve here. Please explain.

Reply:
Thank you very much for that question @Hadeneekeh. You see, when tests get ran, it makes the node_env get set to test, and this made the social login feature not work because of the way it was initially implemented. With this, the issue if node_env remaining as TEST will be solved. Thank you

Reply:
Okay, thanks for the response.

Reply:
Thanks for the observation of the `v`. it was a typo